tests/functional/z_cluster/nodes/test_nodes_maintenance.py

```
@@ -972,7 +972,12 @@ def _run_write_io(self, pod_obj):
                 fio_filename="drain_write_test",
             )
             get_fio_rw_iops(pod_obj)
-        except (CommandFailed, TimeoutExpiredError, TimeoutError) as e:
+        except (
+            CommandFailed,
+            TimeoutExpiredError,
+            TimeoutError,
+            TimeoutExpired,
+        ) as e:
             log.warning(f"Write IO failed: {e}")
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)